keyboards/rhymestone/common/glcdfont.c

```diff
@@ -0,0 +1,244 @@
+// This is the 'classic' fixed-space bitmap font for Adafruit_GFX since 1.0.
+// See gfxfont.h for newer custom bitmap font info.
+
+#ifndef FONT5X7_H
+#define FONT5X7_H
+
+#ifdef __AVR__
+ #include <avr/io.h>
+ #include <avr/pgmspace.h>
+#elif defined(ESP8266)
+ #include <pgmspace.h>
+#else
+ #define PROGMEM
+#endif
+
```

Comment:
You don't need all of this. 
```suggestion
 #pragma once
```

Reply:
#include <avr/pgmspace.h>
Above include was needed.
And isn't needed #pragma once
Thanks,